.github/workflows/ci.yml

```diff
@@ -20,10 +20,6 @@ jobs:
       - uses: actions/checkout@v1
       - name: GitHub Action for SwiftLint
         uses: norio-nomura/action-swiftlint@3.1.0
-      - name: GitHub Action for SwiftLint with --strict
-        uses: norio-nomura/action-swiftlint@3.1.0
-        with:
-          args: --strict
```

Comment:
strict is ignoring the `.swiftlint` config file and redundant to the line 20 after removing strict.